tests/core/evm/vm/test_message.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+import evm.vm.message as m
+
+message_object = m.Message(
+    1,                      #gas
+    1,                      #gas_price
+    b'',                    #to
+    '11111111111111111111', #sender
+    1,                      #value
+    '',                     #data
+    'asdf',                 #code
+    '22222222222222222222'  #origin
+    )                    
+
+def test_initilizes_properly():
+    # all __init__ parameters are properly validated.
```

Comment:
Having a little trouble understanding how to properly _validate_ __init__ parameters without redoing the tests in validation.py.  They're automatically validated when the the class is instantiated.

Can you elaborate a little as to the methodology?